docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,148 @@ It is also possible to configure the scope values that would be used via the sco
 
 NOTE: If both the deployment and subsystem configuration is present, then the subsystem configuration will override the deployment values.
 
+== Sending Request Objects as a JWT
```

Comment:
Sending a Request Object as a JWT

Reply:
Just to check, any reason to not use === here (just to be consistent with the format used for other sections like the scope section for example)

Reply:
I did == to match with sections like Virtual Security or Multi-Tenancy Support. Since this section is quite large it could make sense to give it a separate section. But I am ok with changing it all to be one level lower too.